lib/modules/Logger.js

```diff
@@ -1,6 +1,17 @@
 export default store => next => action => {
-  console.log('dispatching', action);
-  const result = next(action);
-  console.log('next state', store.getState());
+  let result;
+  console.log(action.type);
+  if (console.group) {
```

Comment:
this looks good! I think you're missing the initial `console.group` call though. how about something like:
```es6
let result;
if (console.group) {
  console.group(action.type);
  /* yadda yadda yadda */
  console.groupEnd();
} else {
  /* normal output */
}
```